src/components/organisms/UiList/UiList.stories.ts

```diff
@@ -0,0 +1,436 @@
+import type {
+  Meta,
+  StoryObj,
+} from '@storybook/vue3';
+import {
+  UiList,
+  type ListProps,
+  UiButton,
+  UiCheckbox,
+  UiRadio,
+} from '@index';
+import {
+  getArgTypes,
+  getAttrs,
+} from '@sb/helpers';
+import { actions } from '@storybook/addon-actions';
+import {
+  BasicStories,
+  BasicStoriesSource,
+  ConditionStories,
+  ConditionStoriesSource,
+  ConditionWithButtonStories,
+  ConditionWithButtonStoriesSource,
+  ConditionWithPrimaryButtonStories,
+  ConditionWithPrimaryButtonStoriesSource,
+  IconInHeadingStories,
+  IconInHeadingStoriesSource,
+  AnswerWithCheckboxStories,
+  AnswerWithCheckboxStoriesSource,
+} from './stories';
+
+type ListArgsType = ListProps;
+type ListMetaType = Meta<ListArgsType>;
+type ListStoryType = StoryObj<ListArgsType>;
+
+const {
+  argTypes,
+  args,
+} = getArgTypes(UiList);
+
+const items = [
+  'Painful swallowing',
+  'Stuffy nose',
+  'Sneeze',
+  'Muscle pain',
+  'Runny nose',
+].map((item) => ({
+  label: item,
+  hasSuffix: false,
+}));
+
+const meta = {
+  title: 'Organisms/List',
+  component: UiList,
+  args: {
+    ...args,
+    items,
+  },
+  argTypes: {
+    ...argTypes,
+    items: { control: 'object' },
+  },
+  excludeStories: [ 'HasChildren' ], // TODO: Design for HasChildren
+  parameters: { chromatic: { disableSnapshot: false } },
+} satisfies ListMetaType;
+export default meta;
+
+export const Basic: ListStoryType = {
+  render(args, {
+    name, argTypes,
+  }) {
+    return {
+      name,
+      components: { BasicStories },
+      setup() {
+        const { attrs } = getAttrs(args, argTypes, name);
+        return { attrs };
+      },
+      template: '<BasicStories v-bind="{...attrs}"/>',
+    };
+  },
+};
+Basic.parameters = { docs: { source: { code: BasicStoriesSource } } };
+export const LargeList: ListStoryType = { ...Basic };
+LargeList.args = {
+  items: [
```

Comment:
Maybe this solution can be use:
```
const largeList = [];
for (let i = 0; i < 10; i++) {
  largeList.push(...items);
}

LargeList.args = {
items: largeList,
.....
}
```

WDYT?

Reply:
Thank you for the motivation. I can use `Array.from` 🙈 
```
items: Array.from({length: 100}, (_, i) => items[i%items.length]),
```